model-engine/model_engine_server/api/files_v1.py

```diff
@@ -44,7 +44,7 @@ async def upload_file(
     )
     return await use_case.execute(
         user=auth,
-        filename=file.filename,
+        filename=file.filename or "",
```

Comment:
when does `file.filename` equal None, should we just throw a 4xx in that case (if it is user fault that `file.filename` could be None?) 

maybe this is fine if `filename` doesn't get used as the only part of an identifier actually, tbh I'm not sure though

Reply:
do we know when filename can be None? eg does the fastapi documentation say anything about it